src/kabasuji/Main.java

```diff
@@ -0,0 +1,43 @@
+package kabasuji;
+
+import java.io.IOException;
+
+import com.jfoenix.controls.JFXDecorator;
+
+import javafx.application.Application;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+public class Main extends Application {
+
+  @FXML
+  private StackPane root;
+
+  public void start(Stage stage) {
+    try {
+      root = FXMLLoader.load(getClass().getResource("/resources/fxml/Main.fxml"));
```

Comment:
loading this FXML causes the controller attached to Main.fxml (see Main.fxml:17) to be created. It's constructor is called, and field injection of fields marked as @FXML occurs, and then initialize is called.